src/app/components/search/SearchResultsCards/ArticleCard.js

```diff
@@ -26,56 +31,95 @@ const ArticleCard = ({
+          ) : null }
           <div>
-            <Tooltip arrow title={teamName}>
-              <span>
-                <TeamAvatar team={{ avatar: teamAvatar }} size="30px" />
-              </span>
-            </Tooltip>
+            <SharedItemCardFooter
+              languageCode={languageCode}
+              tags={tags}
+              tagOptions={tagOptions}
+              onChangeTags={onChangeTags}
+            />
           </div>
-        ) : null }
-        <div>
-          <SharedItemCardFooter
-            languageCode={languageCode}
-            tags={tags}
-            tagOptions={tagOptions}
-            onChangeTags={onChangeTags}
-          />
         </div>
-      </div>
-      { (statusLabel || date || variant === 'fact-check') ?
-        <div className={styles.cardRight}>
-          <div className={styles.cardRightTop}>
-            { statusLabel && <ItemRating rating={statusLabel} ratingColor={statusColor} /> }
-            { variant === 'fact-check' && <ItemReportStatus publishedAt={publishedAt ? new Date(publishedAt * 1000) : null} /> }
-          </div>
-          { date &&
-            <ItemDate
-              date={new Date(date * 1000)}
-              tooltipLabel={<FormattedMessage id="factCheckCard.dateLabel" defaultMessage="Published at" description="Date tooltip label for fact-check cards" />}
-            />
-          }
-        </div> : null
-      }
-    </Card>
-  </div>
-);
+        { (statusLabel || date || variant === 'fact-check') ?
+          <div className={styles.cardRight}>
+            <div className={styles.cardRightTop}>
+              { statusLabel && <ItemRating rating={statusLabel} ratingColor={statusColor} /> }
+              { variant === 'fact-check' && <ItemReportStatus publishedAt={publishedAt ? new Date(publishedAt * 1000) : null} /> }
+            </div>
+            { date &&
+              <ItemDate
+                date={new Date(date * 1000)}
+                tooltipLabel={<FormattedMessage id="factCheckCard.dateLabel" defaultMessage="Published at" description="Date tooltip label for fact-check cards" />}
+              />
+            }
+          </div> : null
+        }
+      </Card>
+      {openEdit && variant === 'explainer' && <ExplainerForm
+        onClose={setOpenEdit}
+        team={{ teamTags: tagOptions }}
+        article={{
+          title,
+          summary,
+          language: languageCode,
+          tags,
+          url,
+          date,
+          createdDate,
+          id,
+        }}
+      />}
+      {openEdit && variant === 'fact-check' && <ClaimFactCheckForm
+        onClose={setOpenEdit}
+        team={{ teamTags: tagOptions }}
+        article={{
+          title,
+          summary,
+          language: languageCode,
+          tags,
+          url,
+          date,
+          rating,
+          createdDate,
+          statuses,
+          id,
+          claim_description,
+          publishedAt,
+        }}
+      />}
+    </div>
+  );
```

Comment:
Issue (blocking): I think we should decouple this from the `ArticleCard` component and maybe have these form component mounted elsewhere, probably the `Articles` component, that should manage the `openEdit` state and so on. We should pass a callback from the parent `Articles` to `ArticleCard` `onClick` prop that will trigger the open form state.